### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
So, this is a fairly large PR, it's split by word family which is my default.
I've tried to exclude translation fixes in general (I have a branch called "ignore" which has some of those at the top).

There are some exceptions.

But, in general, the categories of fixes are:
* Comments
* Code
* API
* CSS
* UI Strings
* Brands (Sefaria, Django)
* Bug fixes / things which should be bug fixes but could introduce bugs if something else dealt w/ the case / wasn't expecting this buggy code to do what it was trying to do
* Changes that probably require a lawyer / contacting the authors for permission

Some changes are to files that I'm really unsure about their purpose (especially things in dev/). When I'm sure things should be ignored, I delete them in my `ignore` branch. I'm happy to do more of that if that's the right thing.

In one case, I've deleted a previous person's "this seems like a typo" comment and just corrected the code (css). And in one case, I've inserted a robot readable (since I'm pretty sure the audience is a screen reader) `??` because the item doesn't make sense and I'm not sure what the correct message is.

I'm happy to split this up however requested (within limits, 100+PRs would be a bit much, but 10 PRs with some direction is manageable).

### CODE REVIEW

File: scripts/reset_berakhot.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
 
 with open("../tmp/berakhot_review_links.csv", 'wb') as csvfile:
     writer = csv.writer(csvfile)
-    writer.writerow(["Ref 1", "Ref 2", "Type", "Anchort Text"])
+    writer.writerow(["Ref 1", "Ref 2", "Type", "Anchor Text"])
```
Comment (jsoref) [inline scripts/reset_berakhot.py]: Seems to be a typo...

File: sefaria/model/text.py [hunk 2]
```diff
@@ -4737,10 +4737,10 @@ def get_multi_title_regex_string(self, titles, lang, for_js=False, anchored=Fals
                            + node.after_title_delimiter_re \
                            + node.address_regex(lang, for_js=for_js, match_range=True) + u")"]
 
-            all_interal = u"|".join(regex_components)
-            if all_interal:
+            all_internal = u"|".join(regex_components)
```
Comment (jsoref) [inline sefaria/model/text.py]: I have no idea if this is a correct change

File: sefaria/system/database.py [hunk 3]
```diff
@@ -83,7 +83,7 @@ def ensure_indices():
     db.sheets.ensure_index("assignment_id")
     db.texts.ensure_index("title")
     db.texts.ensure_index([("priority", pymongo.DESCENDING), ("_id", pymongo.ASCENDING)])
-    db.texts.ensure_index([("versionTitle", pymongo.ASCENDING), ("langauge", pymongo.ASCENDING)])
+    db.texts.ensure_index([("versionTitle", pymongo.ASCENDING), ("language", pymongo.ASCENDING)])
```
Comment (jsoref) [inline sefaria/system/database.py]: If this is reliant on an external object, this could break stuff

File: sefaria/views.py [hunk 4]
```diff
@@ -111,14 +111,14 @@ def accounts(request):
 
 def subscribe(request, email):
     """
-    API for subscribg is mailing lists, in `lists` url param.
+    API for subscribe is mailing lists, in `lists` url param.
```
Comment (jsoref) [inline sefaria/views.py]: This line is hard to parse, it's possible that `is` needs to change or that the correction should be `subscribing` or...

File: static/css/s2.css [hunk 5]
```diff
@@ -1337,7 +1337,7 @@ a.readerNavCategory {
 .navButton:active,
 .readerNavCategory:active,
 .refLink:active,
-.resourceseLink {  /* likely a typo in this word but not sure if this this still applies */
+.resourcesLink {
```
Comment (jsoref) [inline static/css/s2.css]: This is the bug fix i called out in the PR description

File: static/css/s2.css [hunk 6]
```diff
@@ -3607,7 +3607,7 @@ body .ui-autocomplete.dictionary-toc-autocomplete .ui-menu-item a.ui-state-focus
   cursor: pointer;
   line-height: 18px;
   color: #000;
-  -webkit-tap-hightlight-color: rgba(0,0,0,0); /* pulled from common.css; should apply more broadly? */
+  -webkit-tap-highlight-color: rgba(0,0,0,0); /* pulled from common.css; should apply more broadly? */
```
Comment (jsoref) [inline static/css/s2.css]: This is a similar bug fix

File: static/css/static.css [hunk 7]
```diff
@@ -1560,7 +1560,7 @@ input#id_email{
     float: left;
 }
 
-#visualGardenPage .control-results-containe{
+#visualGardenPage .control-results-container{
```
Comment (jsoref) [inline static/css/static.css]: And another

File: static/css/static.css [hunk 8]
```diff
@@ -1748,4 +1748,4 @@ body.interface-hebrew #visualGardenPage .filter-title .reset {
 }
 .phone-number{
   unicode-bidi: plaintext;
-}
\ No newline at end of file
+}
```
Comment (jsoref) [inline static/css/static.css]: It would be nice if all text files had a blank line before EOF so that PRs like mine don't trigger this.

File: static/dev/animations.html [hunk 9]
```diff
@@ -1289,7 +1289,7 @@
 				response.toVerse = cv[1]
 			} else if (cv.length == 1) {
 				response.toVerse = cv[0]
-				response.toChaptere = response.chapter
+				response.toChapter = response.chapter
```
Comment (jsoref) [inline static/dev/animations.html]: This could be a bug fix (or as noted elsewhere introduce a bug if something wasn't expecting this code to function)

File: static/dev/archive/get.html [hunk 10]
```diff
@@ -49,7 +49,7 @@
 	<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.8.7/jquery-ui.min.js"></script>
 	<script type="text/javascript">
 	
-	books = ["Amos", "Chronicles 1", "Chronicles 2", "Daniel", "Deuteronomy", "Ecclesiastes", "Esther", "Exodus", "Ezekiel", "Ezra Nehemiah", "Genesis", "Habakkuk", "Haggai", "Hosea", "Isaiah", "Jeremiah", "Job", "Joel", "Jonah", "Joshua", "Judges", "Kings 1", "Kings 2", "Lamenstaions", "Leviticus", "Malachi", "Nahum", "Numbers", "Obadiah", "Proverbs", "Psalms", "Ruth", "Samuel 1", "Samuel 2", "Song of Songs", "Zechariah", "Zephaniah"]
+	books = ["Amos", "Chronicles 1", "Chronicles 2", "Daniel", "Deuteronomy", "Ecclesiastes", "Esther", "Exodus", "Ezekiel", "Ezra Nehemiah", "Genesis", "Habakkuk", "Haggai", "Hosea", "Isaiah", "Jeremiah", "Job", "Joel", "Jonah", "Joshua", "Judges", "Kings 1", "Kings 2", "Lamentations", "Leviticus", "Malachi", "Nahum", "Numbers", "Obadiah", "Proverbs", "Psalms", "Ruth", "Samuel 1", "Samuel 2", "Song of Songs", "Zechariah", "Zephaniah"]
```
Comment (jsoref) [inline static/dev/archive/get.html]: This is the odd file I highlighted initially, I normally don't fix misspellings, but this seemed egregious, and perhaps the file doesn't need to live...

File: static/js/AddToSourceSheetBox.jsx [hunk 11]
```diff
@@ -152,7 +152,7 @@ class AddToSourceSheetBox extends Component {
         <div className="dropdown">
           <div className="dropdownMain noselect" onClick={this.toggleSheetList}>
             <i className="dropdownOpenButton noselect fa fa-caret-down"></i>
-            {this.state.sheetsLoaded ? (this.state.selectedSheet.title === null ? Sefaria._("Untitled Source Sheet") : this.state.selectedSheet.title.stripHtml()) : <LoadingMessage messsage="Loading your sheets..." heMessage="טוען את דפי המקורות שלך"/>}          </div>
+            {this.state.sheetsLoaded ? (this.state.selectedSheet.title === null ? Sefaria._("Untitled Source Sheet") : this.state.selectedSheet.title.stripHtml()) : <LoadingMessage message="Loading your sheets..." heMessage="טוען את דפי המקורות שלך"/>}          </div>
```
Comment (jsoref) [inline static/js/AddToSourceSheetBox.jsx]: This could be a bug fix

File: static/js/Header.jsx [hunk 12]
```diff
@@ -328,7 +328,7 @@ Header.propTypes = {
   registerAvailableFilters:    PropTypes.func,
   setUnreadNotificationsCount: PropTypes.func,
   handleInAppLinkClick:        PropTypes.func,
-  headerMesssage:              PropTypes.string,
+  headerMessage:              PropTypes.string,
```
Comment (jsoref) [inline static/js/Header.jsx]: I generally don't spend time worrying about whitespace (my tooling doesn't show it to me), but I can fix it later if requested.

File: static/js/Misc.jsx [hunk 13]
```diff
@@ -254,7 +254,7 @@ class ToggleSet extends Component {
                 key={option.name}
                 set={this.props.name}
                 role={option.role}
-                ariaLable={option.ariaLabel}
+                ariaLabel={option.ariaLabel}
```
Comment (jsoref) [inline static/js/Misc.jsx]: This could be a bug fix...

File: static/js/ReaderPanel.jsx [hunk 14]
```diff
@@ -1258,7 +1258,7 @@ class ReaderDisplayOptionsMenu extends Component {
           settings={this.props.settings} />);
 
     var layoutOptions = [
-      {name: "continuous", fa: "align-justify", role: "radio", ariaLabel: "Show Text as a paragram" },
+      {name: "continuous", fa: "align-justify", role: "radio", ariaLabel: "Show Text as a paragram (?! what ??)" },
```
Comment (jsoref) [inline static/js/ReaderPanel.jsx]: This is the thing I called out as being pretty odd/needing a look. Obviously it shouldn't be merged as-is.

File: static/js/TextRange.jsx [hunk 15]
```diff
@@ -241,15 +241,15 @@ class TextRange extends Component {
     $(event.target).closest("sup").next("i.footnote").toggle();
     this.placeSegmentNumbers();
   }
-  parashahHeader(data, segment, includeAliyout=false) {
-    // Returns the English/Hebrew title of a Parasha, if `ref` is the beginning of a new parahsah
+  parashahHeader(data, segment, includeAliyot=false) {
```
Comment (jsoref) [inline static/js/TextRange.jsx]: Aliyot is used a lot, this odd spelling was a local thing which I couldn't justify.

File: static/js/VersionBlock.jsx [hunk 16]
```diff
@@ -180,8 +180,8 @@ class VersionBlock extends Component {
               {licenses.map(v => <option key={v} value={v}>{v?v:"(None Listed)"}</option>)}
             </select>
 
-            <label id="digitzedBySefaria_label" htmlFor="digitzedBySefaria">Digitized by Sefaria</label>
-            <input type="checkbox" id="digitzedBySefaria" checked={this.state.digitizedBySefaria} onChange={this.onDigitizedBySefariaChange}/>
+            <label id="digitizedBySefaria_label" htmlFor="digitizedBySefaria">Digitized by Sefaria</label>
```
Comment (jsoref) [inline static/js/VersionBlock.jsx]: Potentially an API break if something external to sefaria relies on this id...

File: static/js/s1/editor.js [hunk 17]
```diff
@@ -29,21 +29,21 @@ $.extend(sjs,  {
...
+	_verseHeights: [],  // stored list of the top position of each verse
 	_scrollMap: []      // stored list of the window top position that should correspond to highlighting each verse
 });
 
 sjs.cache.params({notes: 1, sheets: 1}); // Default parameters to getting texts.
 
 sjs.ratySettings = { // for text review ratings
 	path: "/static/img/raty/",
-	hints: ["Major problems", "Some problems", "Seems good", "Good", "Definately good"]
+	hints: ["Major problems", "Some problems", "Seems good", "Good", "Definitely good"]
```
Comment (jsoref) [inline static/js/s1/editor.js]: I suspect humans can see this

File: static/js/s1/editor.js [hunk 18]
```diff
@@ -983,7 +983,7 @@ $(function() {
 
 		var n = sjs.selected_verses[0];
 		var top = $(".segmentLabel").eq(n-1).position().top - 100;
-		$("html, body").animate({scrollTop: top, duation: 200});
+		$("html, body").animate({scrollTop: top, duration: 200});
```
Comment (jsoref) [inline static/js/s1/editor.js]: bugfix?

File: static/js/s1/editor.js [hunk 19]
```diff
@@ -1541,7 +1541,7 @@ sjs.lexicon = {
 							"Jastrow<span class='ui-icon ui-icon-extlink'></span></a>" : "") +
 						"</div>";
 				}
-				html += (sense.source === "CAL Lexicon" ? "<i class='definitionSource'>Definitions courtesry of <a href='http://cal1.cn.huc.edu/browseheaders.php?first3=" + entry.term + "' target='_blank'>" +
+				html += (sense.source === "CAL Lexicon" ? "<i class='definitionSource'>Definitions courtesy of <a href='http://cal1.cn.huc.edu/browseheaders.php?first3=" + entry.term + "' target='_blank'>" +
```
Comment (jsoref) [inline static/js/s1/editor.js]: I don't think this needs a lawyer

File: static/js/s1/editor.js [hunk 20]
```diff
@@ -2586,7 +2586,7 @@ function aboutHtml(data) {
 function updateVisible() {
 	// Update view based on what text is currently visible in the viewport.
 	// Currently, this means scrolling the commentary box to sync with content
-	// visible in the baesetext.
+	// visible in the basetext.
```
Comment (jsoref) [inline static/js/s1/editor.js]: probably right, but sometimes words/tokens refer to hebrew, so possibly wrong

File: static/js/s1/util.js [hunk 21]
```diff
@@ -267,7 +267,7 @@ sjs.alert = {
 		sjs.alert._show(alertHtml);
 	},
 	loadingSidebar: function() {
-		sjs._$commentaryViewPort.removeClass("noCommenatary")
+		sjs._$commentaryViewPort.removeClass("noCommentary")
```
Comment (jsoref) [inline static/js/s1/util.js]: Bugfix...

File: templates/activity.html [hunk 22]
```diff
@@ -14,9 +14,9 @@
 
 {% block description %}
 	{% if single %}
-{% trans "View the complete revision hisotry of this text on Sefaria." %}
+{% trans "View the complete revision history of this text on Sefaria." %}
```
Comment (jsoref) [inline templates/activity.html]: Translations called out at the beginning

File: templates/static/educators.html [hunk 23]
```diff
@@ -97,7 +97,7 @@ <h2>
 
                 </div>
                 <div class="imageBox">
-                    <img src="{% static 'img/partnership-initiative-image.png' %}" alt="Image of teachers at a Sefaria educator partnership initative presentation" />
+                    <img src="{% static 'img/partnership-initiative-image.png' %}" alt="Image of teachers at a Sefaria educator partnership initiative presentation" />
```
Comment (jsoref) [inline templates/static/educators.html]: I'd hope these two changes aren't objectionable

**Comment by edamboritz** [comment]
Thank you for contributing all these spelling corrections! 
We'll just merge them in to a branch and test that nothing breaks accidentally.
